docs/pages/badge.js

```diff
@@ -274,7 +274,7 @@ function ModuleExample() {
   return (
     <Box column={12} maxWidth={800} padding={2}>
       <Module
-        badgeText="Beta"
+        badge={{text: "Beta"}}
```

Comment:
nittest of nits: this code doesn't get Prettier-ified but this should include spaces and use single quotes, like `{{ text: 'Beta' }}` (here and throughout)